types/katex/index.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference lib="dom" />
```

Comment:
Why set this here? Typically users are expected to set this in their tsconfig, not have some random library auto load the DOM types.